docs/manual.txt

```diff
@@ -1351,6 +1351,10 @@ typogrify_sans_widont
 minify_lines
    **THIS FILTER HAS BEEN TURNED INTO A NOOP** and currently does nothing.
 
+normalize_html
```

Comment:
No, _normalize_html is internal, works on strings, the one without the underscore is a proper filter that works on files :-)